crypto/stack/stack.c

```diff
@@ -293,7 +322,7 @@ void *OPENSSL_sk_shift(OPENSSL_STACK *st)
         return (NULL);
     if (st->num <= 0)
         return (NULL);
```

Comment:
While you're at it, why not fix styling? I mean parenthesis around return value.